### PR TITLE
Dot Net Assignment 3

### PR DESCRIPTION
**Pull request by emtec-damodhar-jadhao**
Minimal api Assignment Completed using ASP.Net Minimal Web API

### CODE REVIEW

File: Minimal_API/Program.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        case "/":
+            return Results.Json(calc.N1 / calc.N2);
+    }
+    return Results.Json("Opration not supported");
+});
+
+
+app.Run();
+
+class Calculator
```
Comment (nitish-poojari) [inline Minimal_API/Program.cs]: Move to another file

Comment (emtec-damodhar-jadhao) [reply Minimal_API/Program.cs]: Moved the Calculator class to another file.

File: Minimal_API/bin/Debug/net6.0/Minimal_API.deps.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    "signature": ""
+  },
+  "compilationOptions": {},
+  "targets": {
+    ".NETCoreApp,Version=v6.0": {
+      "Minimal_API/1.0.0": {
+        "dependencies": {
+          "Swashbuckle.AspNetCore": "6.2.3"
+        },
+        "runtime": {
```
Comment (nitish-poojari) [inline Minimal_API/bin/Debug/net6.0/Minimal_API.deps.json]: Always exclude Bin and Obj folders from PR

**Review by nitish-poojari — APPROVED** [review]

File: Minimal_API/bin/Debug/net6.0/Minimal_API.deps.json (hunk 2 above)
Comment (emtec-damodhar-jadhao) [reply Minimal_API/bin/Debug/net6.0/Minimal_API.deps.json]: Deleted bin and obj form the project

**Pull request merged**